winml/lib/Api/LearningModelBinding.cpp

```diff
@@ -487,7 +485,6 @@ STDMETHODIMP LearningModelBinding::Bind(
     UINT32 cchName,
     IUnknown* value) {
   try {
-    _winmlt::TelemetryEvent binding_event(_winmlt::EventCategory::kBinding);
```

Comment:
>binding_event [](start = 28, length = 13)

I think it would be useful to still have a ETW tracelogging write event that indicates that a binding event in WinML for debugging purposes so I don't this should be removed.